src/components/ReportActionItem/TaskView.js

```diff
@@ -53,94 +55,104 @@ function TaskView(props) {
     const disableState = !canEdit || !isOpen;
     return (
         <View>
-            <Hoverable>
-                {(hovered) => (
-                    <PressableWithSecondaryInteraction
-                        onPress={Session.checkIfActionIsAllowed((e) => {
-                            if (e && e.type === 'click') {
-                                e.currentTarget.blur();
-                            }
+            <OfflineWithFeedback
+                shouldShowErrorMessages
+                errors={lodashGet(props, 'report.errorFields')}
```

Comment:
The current code causes a console error. We need to get the errors from `report.errorFields.editTask`, which is an object of error messages keyed by microtime. For example:
```
Onyx.merge('report_7333847874765172', {
    errorFields: {
        editTask: {
            123423: "Unexpected error",
        },
    },
});
```